NOTICE

```diff
@@ -0,0 +1,5 @@
+Apache Iceberg
```

Comment:
I'm not sure do we need to use `Apache Iceberg Rust` here?

Reply:
Yes, I think that would be best

Reply:
Fixed.